src/lib/free-queue/free-queue.js

```diff
@@ -249,4 +246,4 @@ export {
   MAX_CHANNEL_COUNT,
   RENDER_QUANTUM_FRAMES,
   FreeQueue
-};
\ No newline at end of file
+};
```

Comment:
Nit: is it possible to keep this unchanged (that is, don't add an additional newline)?